### PR TITLE
AutoUpdate Feature Health Data Logging Changes

### PR DESCRIPTION
**Pull request by niteskum**
This PR includes changes for Auto Update Feature Health Data Logging Changes

### CODE REVIEW

**Comment by navch** [comment]
Can you please add some description, why we doing this?

File: installer/mac/update.sh [hunk 1]
```diff
@@ -176,13 +176,13 @@ function updateBrackets(){
         exitStatus=$?
         if [ $exitStatus -ne 0 ]
         then
-            printError "Brackets could not be opened. open returnd $exitStatus."
+            printError "Brackets could not be opened. open returnd $exitStatus : INS_12"
         else
             printInfo "Opened Brackets."
-        fi
+        fiS
```
Comment (navch) [inline installer/mac/update.sh]: Is this intentional?

File: installer/mac/update.sh [hunk 2]
```diff
@@ -38,37 +38,37 @@ function printError(){
 function verifySetup(){
     if [ "$APP_DIR" == "$EMPTY_STRING" ]
     then
-        printError "Brackets app dir is missing"
+        printError "Brackets app dir is missing: INS_O1"
```
Comment (navch) [inline installer/mac/update.sh]: Don't we need "Error" word in these strings?

Comment (niteskum) [reply installer/mac/update.sh]: "ERROR:" String will be added by printError Function so we don't need to add explicitly.

**Comment by nethip** [comment]
Thanks @niteskum for the changes.

**Pull request merged**